frontend/src/components/AdminReport.tsx

```diff
@@ -220,71 +243,124 @@ const AdminReport = () => {
 
     const buttonsContainerStyle = {
       display: 'flex',
-      justifyContent: 'center', // Aligns child elements (buttons) in the center
+      justifyContent: 'center',
       gap: '8px',
-      marginTop: '20px' // Adds some space between the buttons and other elements above
+      marginTop: '20px'
     };
 
-    const dropdownStyle = {
-      backgroundColor: '#007BFF',
+    const cancelButtonStyle = {
+      backgroundColor: 'white',
+      color: 'black',
```

Comment:
Change the color to okb_colors.dark_gray in colors.ts. The okb_colors import is already at the top of the file.